scripts/cli/src/helpers/asset_helper.ts

```diff
@@ -76,6 +77,6 @@ export async function assetBalance(
   did: IdentityId
 ): Promise<number> {
   const api = await ApiSingleton.getInstance();
-  const balance = (await api.query.asset.balanceOf(ticker, did)).toNumber();
+  const balance = u8aToBn((await api.query.asset.balanceOf(ticker, did)).toU8a()).toNumber();
```

Comment:
There are still at least 22 usages of `u8aToBn`.  Please go through and remove these hacks.